library/src/morpheus-master/sass/modules/_toolmenu.scss

```diff
@@ -396,8 +396,6 @@ nav#subSites{
 				}
 
 				&:before{
```

Comment:
If you are removing two lines from this rule, you might as well remove the whole rule because the need for this rule is to add the transparent border placeholder to the elements.